spec/filters/syslog_pri_spec.rb

```diff
@@ -1,5 +1,44 @@
-require "logstash/devutils/rspec/spec_helper"
-require 'logstash/filters/syslog_pri'
+# encoding: utf-8
+require_relative "../spec_helper"
+require "logstash/plugin"
+require "logstash/event"
 
 describe LogStash::Filters::Syslog_pri do
+
+  subject          { LogStash::Filters::Syslog_pri.new( "syslog_pri_field_name" => "syslog_pri" ) }
+  let(:properties) { {:name => "foo" } }
+  let(:event)      { LogStash::Event.new(properties) }
+
+  it "should register without errors" do
+    plugin = LogStash::Plugin.lookup("filter", "syslog_pri").new( "facility_labels" => ["kernel"] )
+    expect { plugin.register }.to_not raise_error
+  end
+
+  describe "filtering" do
+
+    let(:msg) { "<1>Sep 20 02:58:12 porridge3 puppet-master[27025]: Compiled catalog for bigbopper.adm.intranet in environment production in 1.31 seconds\n" }
```

Comment:
@suyograo my syslog knowledge is very low, so I would try to look for message with this priority/facility examples, otherwise, can you provide me with them? 

I will also check the default.

Thanks a lot for your time.